<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionForm</class>
 <widget class="QWidget" name="QuestionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加/修改试题</string>
  </property>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>试题名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Content">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>试题题干：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Type">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>试题类型：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ok">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_limit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>限填10个字</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Content">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>370</width>
     <height>100</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Type">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_Creator">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>出题人：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Creator">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_Answer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>试题答案：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Difficulty">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>试题难度：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Difficulty">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_Difficulty_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>注意:数值越大难度越大</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Keyword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关键字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Keyword">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Bool">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>370</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_Yes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>对</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_No">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>错</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Single">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>370</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_A">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_B">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_C">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_D">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_E">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Multi">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>370</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="checkBox_A">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_B">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_C">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_D">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_E">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Blank">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>370</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBox_Num">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Blank">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>共有</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Blank_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>项需填空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_yes">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_no">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重填</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1、</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2、</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3、</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4、</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_Answer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>370</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_IsText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>270</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>是纯文本试题，不钩选为其他类型试题</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改非文本题干</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>370</width>
     <height>100</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
